app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -137,11 +129,10 @@ public void changeProgressBarVisibility(boolean isVisible) {
     }
 
     /**
-     * Clears sync message displayed with progress bar before contributions list became visible
+     * Shows welcome message if user has no contributions yet i.e. new user.
      */
-    protected void clearSyncMessage() {
-        waitingMessage.setVisibility(GONE);
-        noDataYet.setVisibility(GONE);
+    protected void maybeShowWelcomeTip(boolean noContributions) {
```

Comment:
Minor Detail: removing the maybe would help shorten the method name. I think "showWelcomeTip(boolean)" is self explanatory enough to indicate that it's a method that decides whether or not to show a welcome tip.